api/src/main/java/io/opencensus/stats/AggregationData.java

```diff
@@ -464,6 +465,24 @@ public static Exemplar create(
         return new AutoValue_AggregationData_DistributionData_Exemplar(
             value, timestamp, attachmentsCopy);
       }
+
+      private static final Comparator<Exemplar> EXEMPLAR_COMPARATOR =
+          new Comparator<Exemplar>() {
+            @Override
+            public int compare(Exemplar o1, Exemplar o2) {
+              return Double.compare(o1.getValue(), o2.getValue());
+            }
+          };
+
+      /**
+       * Returns a {@link Comparator} of {@link Exemplar} that compares the double value.
+       *
+       * @return a {@code Comparator} of {@code Exemplar}
+       * @since 0.16
+       */
+      public static Comparator<Exemplar> getComparator() {
+        return EXEMPLAR_COMPARATOR;
```

Comment:
Exporters need to be Java 7 compatible so method reference is not available. Is there a concise way to compare objects in Java 7?

Reply:
For multiple reasons we need utils for all exporters that we tend to put in the public api. Maybe we should have a metrics-exporter-common package for this kind of things. Also for example the thread logic in stats can be moved there as abstract class or something.

What do you think?

Reply:
SGTM. For this particular util I think it's more appropriate to put it under `exemplar-util` since it's possible to be used in user applications too.

Reply:
> For multiple reasons we need utils for all exporters that we tend to put in the public api. Maybe we should have a metrics-exporter-common package for this kind of things. Also for example the thread logic in stats can be moved there as abstract class or something.
> 
> What do you think?

If we end up needing many (or complex) shared utilities for exporters then I agree that we should add a separate artifact, especially if it will be useful for exporters written outside of this repository.

> SGTM. For this particular util I think it's more appropriate to put it under exemplar-util since it's possible to be used in user applications too.

I agree that exemplar-util is the best place for this method.  I'm still not sure that it's worth adding a public method, though, since it is relatively simple, and it doesn't add functionality that couldn't be implemented directly in the exporter.  Which exporters will need to do this type of comparison?

> Exporters need to be Java 7 compatible so method reference is not available. Is there a concise way to compare objects in Java 7?

I meant that the method will be less useful once the project moves to Java 8.  I don't know of a concise way to compare objects in Java 7, except possibly with Guava's Ordering.

Reply:
> Which exporters will need to do this type of comparison?

Currently only Stackdriver, possibly Prometheus and Datadog later.

> I meant that the method will be less useful once the project moves to Java 8.

If this is the case, I agree this would be simple enough to be handled in the exporter/client code.